util/envDynawo.sh

```diff
@@ -1535,23 +1525,13 @@ find_lib_system_path() {
   if [ ! -f "$DYNAWO_INSTALL_DIR/bin/dynawo" ]; then
     error_exit "Dynawo binary should exist to deploy and find system library used to link against it."
   fi
-  if [ "$DYNAWO_LIBRARY_TYPE" = "SHARED" ]; then
-    if [ "`uname`" = "Linux" ]; then
-      path=$(ldd $DYNAWO_INSTALL_DIR/bin/dynawo | grep "$1" | cut -d '>' -f 2 | awk '{print $1}' | sed "s/lib$1.*//g"  | uniq)
-    elif [ "`uname`" = "Darwin" ]; then
-      path=$(otool -L $DYNAWO_INSTALL_DIR/bin/dynawo | grep "$1" | awk '{print $1}' | sed "s/lib$1.*//g"  | uniq)
-    else
-      echo "OS not supported."
-      exit 1
-    fi
-  elif [ "$DYNAWO_LIBRARY_TYPE" = "STATIC" ]; then
-    LIBRARY_SUFFIX="a"
-    if [ ! -f "$DYNAWO_BUILD_DIR/CMakeCache.txt" ]; then
-      error_exit "$DYNAWO_BUILD_DIR should not be deleted before deploy to be able to determine include system paths used during compilation."
-    fi
-    path=$(cat $DYNAWO_BUILD_DIR/CMakeCache.txt | grep -i "${1}_LIBRARY_DIR_RELEASE:PATH" | cut -d '=' -f 2)
+  if [ "`uname`" = "Linux" ]; then
+    path=$(ldd $DYNAWO_INSTALL_DIR/bin/dynawo | grep "$1" | cut -d '>' -f 2 | awk '{print $1}' | sed "s/lib$1.*//g"  | uniq)
+  elif [ "`uname`" = "Darwin" ]; then
```

Comment:
As Darwin is Apple specific I think we can also drop this branch of the if

Reply:
OK